tools/annotation_tools/turk_with_s3/run_tool_1B.py

```diff
@@ -26,7 +26,7 @@
 
 # Load input commands and create a separate HIT for each row
 rc = subprocess.call(
-    ["python3 create_jobs.py --xml_file fetch_question_B.xml --tool_num 2 --input_csv B/turk_input.csv"], shell=True
+    ["python3 create_jobs.py --xml_file fetch_question_B.xml --tool_num 2 --input_csv B/turk_input.csv --job_spec_csv B/turk_job_specs.csv"], shell=True
```

Comment:
nit: why is there an inconsistency between `python` and `python3` in this file ? Can we use one ?